### DISCUSSION TITLE
The AboveStory Tag and KaTeX Macros

### DISCUSSION THREAD

**Discussion by kookma** [TiddlyWiki/TiddlyWiki5#6030] [Q&A]
Referring to https://github.com/Jermolene/TiddlyWiki5/pull/5933

I have recently realized if a tiddler contains KaTeX macros and tagged with `$:/tags/AboveStory` all macros will be accessible in all tiddlers see as an example: https://mathematik.tiddlyspot.com/#%24%3A%2F_macros%2FKaTeX

My questions are:
- do you recommend such a solution?
- how much overhead a tiddler tagged with `$:/tags/AboveStory` does apply to Tiddlywiki?

**Comment by pmario** [comment]
It should have the same overhead as every other macro tagged $:/tags/Macro.

**Comment by linonetwo** [comment]
I think this will make it difficult to be found by snippet user, for example, codemirror-enhanced or commandpalette, because they are using `$:/tags/Macro` to find out all macro snippets.
